runtime/parser/parse_metrics_view.go

```diff
@@ -568,6 +597,22 @@ func (p *Parser) parseMetricsView(node *Node) error {
 		node.Refs = append(node.Refs, ResourceName{Kind: ResourceKindTheme, Name: tmp.DefaultTheme})
 	}
 
+	// Add annotations as refs to the end of the metrics view.
+	for _, annotation := range tmp.Annotations {
```

Comment:
There are three loops over `tmp.Annotations` – looks like the first two loops can be combined into one here